public-docs/installation-reaction-platform.md

```diff
@@ -0,0 +1,104 @@
+---
+id: installation-reaction-platform
+title: Install with Reaction Platform
+---
+
+Follow these instructions to run the entire Reaction Platform. 
+
+The Platform will install and run the the entire suite of Reaction services in the these directories:
+
+| Directory                                                                                  | Services                        |
+| ------------------------------------------------------------------------------------------ | ------------------------------- |
+| [`reaction`](https://github.com/reactioncommerce/reaction)                                 | GraphQL API, Classic UI, Mongo  |
+| [`reaction-hydra`](https://github.com/reactioncommerce/reaction-hydra)                     | Authentication server, Postgres |
+| [`reaction-next-starterkit`](https://github.com/reactioncommerce/reaction-next-starterkit) | Next.js storefront application  |
+
+## Installation
+
+1. Requirements:
+
+- Install Docker and Docker Compose ([Mac](https://docs.docker.com/docker-for-mac/install/) | [Windows](https://docs.docker.com/docker-for-windows/install/)* | [Linux](https://docs.docker.com/compose/install/#install-compose)**) 
+- Install [Node.js](https://nodejs.org/en/)
+- A GitHub account with a configured SSH key
+
+> *Windows: Reaction Platform has not been fully tested in Windows at this time.
+> **Linux: Docker Compose is included when installing Docker on Mac and Windows, but will need to be installed separately on Linux.
+
+1. Before you get started, Make sure you are not running any applications the necessary ports,  `3000` and `4000`.
+
+2. Clone [**Reaction Platform**](https://github.com/reactioncommerce/reaction-platform)
+
+```sh
+git clone git@github.com:reactioncommerce/reaction-platform.git
+cd reaction-platform
+```
+
+4. Now you're ready to install. Run this command to bootstrap and start all of the services:
+
+```sh
+make
+```
+
+This process may take some time. The Platform is checking that dependencies are present, cloning the sub-projects from GitHub, downloading and building Docker images, and starting services.
+
+5. As the various repositories are downloading and services are starting, check the logs by running `docker-compose logs -f` in the specific repository. For example, to display the `reaction` logs, run:
```

Comment:
@ticean Wasn't sure what the best step was here. I want something to include a step to help users understand that there is something going on behind the scenes, and give them ways to figure out what is going on behind the scenes. I was also thinking of showing what the logs should look like when they know they are ready.. like:

---
5. Once you see that the services started, you can start viewing the logs for each service:

```
Starting reaction-hydra_postgres_1 ... done
Starting reaction-hydra_hydra-migrate_1 ... done
Starting reaction-hydra_hydra_1         ... done
reaction_mongo_1 is up-to-date
Starting reaction_mongo-init-replica_1 ... done
Starting reaction_reaction_1           ... done
Starting reaction-next-starterkit_web_1 ... done
```